.github/workflows/deploy.yml

```diff
@@ -46,6 +50,13 @@ jobs:
       url: ${{ steps.deploy-to-webapp.outputs.webapp-url }}
 
     steps:
+      - name: Az CLI login
+        uses: azure/login@v1
+        with:
+          client-id: d1121761-5bcf-486b-aff0-e6451b4f4e05
+          tenant-id: 398a6654-997b-47e9-b12b-9515b896b4de
+          subscription-id: 550eb99d-d0c7-4651-a337-f53fa6520c4f
```

Comment:
Can we make these repo secrets please? Like in [this example](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-azure)?

Reply:
This is also the approach that was recommended in the other - now closed - PR: https://github.com/primer/primer.style/pull/363

Reply:
Hmm we don't have them as secrets in primer/view_components. I don't really understand why they need to be kept secret. @stephanmiehe even sent them to me in Slack and acknowledged they aren't secret.

Reply:
Okay, I read these as potential - albeit low risk - attack vectors, but as long as SecOps are okay with it 👍 

FYI I can see in @stephanmiehe's [pr](https://github.com/primer/primer.style/pull/363/files#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3R61) that they are inserted as secrets. Flagging as it's a very public repo, so prefer to be safe than sorry.